scripts/deploy.sh

```diff
@@ -27,6 +27,12 @@ function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
 
+apt-get install python-dev -y
+curl https://raw.githubusercontent.com/pypa/pip/7.1.2/contrib/get-pip.py -o /opt/get-pip.py
```

Comment:
I'd like to point out that pinning to 7.1.2 here does not guarantee that pip 7.1.2 will be installed. All that guarantees is that pip 7.1.2 is encoded into the file so it can download the latest pip. If this host isn't locked down with a specific repository (that has a pinned version of pip) then using get-pip.py will get whatever the latest is.